### PR TITLE
Notification page

### PR DESCRIPTION
**Pull request by larajanecka**
Added notification page to organizer for managing then notifications at the top of the home page of the attendeeapp

@Calligre/frontend 
@schneider-clarisse  I added a classname tr-expired to rows where the notification has expired if you wanted to style these in a particular way.

### CODE REVIEW

File: src/js/pages/organizer/Notifications.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    row.forEach(id => NotificationStore.delete(id));
+  }
+
+  formatExpiry = (cell) => {
+    const format = 'MMM Do hh:mm a';
+    return moment(cell).format(format);
+  }
+
+  insertModal = (columns, validateState, ignoreEditable) => {
+    return (<InsertModalBody columns={columns} validateState={validateState} ignoreEditable={ignoreEditable} />);
```
Comment (schneider-clarisse) [inline src/js/pages/organizer/Notifications.jsx:63]: discussed offline, not quite necessary, could follow calendar page

**Review by schneider-clarisse — CHANGES_REQUESTED** [review]

File: src/js/pages/organizer/Notifications.jsx (hunk 1 above)
Comment (larajanecka) [reply src/js/pages/organizer/Notifications.jsx:63]: The calendar table doesnt work so I'm going to leave this as is at look at fixing the calendar table to match it.

**Comment by StevenLocke** [comment]
The dates in the table seem off, with 4/5 of them coming from 1970. More than likely a unix conversion error.

**Comment by larajanecka** [comment]
I'll check it out, looks like a millisecond issue

**Pull request merged**